requirements.txt

```diff
@@ -1 +1 @@
-tf-nightly
\ No newline at end of file
+tensorflow==2.1.0
```

Comment:
How would we handle this if #864 goes through? Make this file into `requirements-test.txt`?